draft-ietf-teep-architecture.md

```diff
@@ -914,6 +914,42 @@ the TAM.
 
 ## TEEP Broker Implementation Consideration
 
+As depicted in {{broker-models}}, there are multiple ways in which a TEEP Broker
+can be implemented, with more or fewer layers being inside the TEE.  For example, in
+model A, the model with the smallest TEE footprint, only the TEEP implementation is inside
+the TEE, whereas the TEEP/HTTP implementation is in the TEEP Broker outside the TEE.
+
+~~~~
+                        Model:    A      B      C     ...
+
+                                 TEE    TEE    TEE
+     +----------------+           |      |      |
+     |      TEEP      |     Agent |      |      | Agent
+     | implementation |           |      |      |
+     +----------------+           v      |      |
+              |                          |      |
+     +----------------+           ^      |      |
+     |    TEEP/HTTP   |    Broker |      |      |
+     | implementation |           |      |      |
+     +----------------+           |      v      |
+              |                   |             |
+     +----------------+           |      ^      |
+     |      HTTP      |           |      |      |
+     | implementation |           |      |      |
+     +----------------+           |      |      v
+              |                   |      |
+     +----------------+           |      |      ^
+     |   TCP or QUIC  |           |      |      | Broker
+     | implementation |           |      |      |
+     +----------------+           |      |      |
+                                 REE    REE    REE
+~~~~
+{: #broker-models title="TEEP Broker Models"}
+
+In other models, additional layers are moved into the TEE, increasing the TEE footprint,
+with the Broker either containing or calling the topmost protocol layer outside of the TEE.
+An implementation is free to choose any of these models.
+
    TEEP Broker implementers should consider methods of distribution, scope and
    concurrency on devices and runtime options.
    Several non-exhaustive options are discussed below. 
```

Comment:
"Several non-exhaustive options are discussed below."

The next paragraph talks about APIs. Was the text in the transport doc? Maybe this line should be removed, or better copy the text over here.

Reply:
Agreed, removing sentence.